carousel/src/carousel/style.css

```
@@ -0,0 +1,44 @@
+.wp-block-wpcomsp-carousel {
+	--animation-speed: 0.3s;
+	--item-gap: var(--wp--preset--spacing--20);
+	--track-height: 430px;
+	position: relative;
```

Stylelint (17.3.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)